### PR TITLE
PS-633 Travis CI integration for QA

### PR DESCRIPTION
**Pull request by holingpoon**
URL:  http://nypl-global-search-qa.us-east-1.elasticbeanstalk.com/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,24 @@ before_script: npm run dist
...
 before_deploy: echo 'All unit tests passed; Successfully built distribution assets;
   Preparing to deploy NYPL Global Search to AWS'
+# For every branch mentioned below that does not match the current branch, the following
+# message would appear:
+#   "Skipping a deployment with the elasticbeanstalk provider because this branch is not permitted"
+# When a Pull Request is submitted and Travis finishes building, the following message would appear:
+#   "Skipping a deployment with the elasticbeanstalk provider because the current build is a pull request."
 deploy:
+- provider: elasticbeanstalk
+  skip_cleanup: true
```
Comment (EdwinGuzman) [inline .travis.yml]: You have `skip_cleanup` twice.

**Comment by EdwinGuzman** [comment]
Just curious, why is this on `qa` and merging into `master`? Shouldn't it be `development` into `qa` and then `master`?

**Comment by holingpoon** [comment]
You are correct. I have the configuration on `qa` because I need to test Travis integration and AWS deployment. I can certainly move the code into `development` and make another pull request for `development` to merge into `qa`.

**Comment by holingpoon** [comment]
Creating a new PR to replace this one.

**Pull request closed without merging**